7-yoda/1-example.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class Order
+{
+    public function updateOrder()
+    {
+        // Typical conditional structure
+        if ( $postType == 'post' ) {
+            // do stuff
+        }
+
+        // This is a typo, since it should be '==' and not '='
+        if ( $postType = 'post' ) {
+            // do stuff
+        }
+
+        // Yoda conditions
+        if ( 'post' == $postType ) {
+            // do stuff
+        }
+
+
+        // Examples
+
+        // This method should not use Yoda conditions
```

Comment:
I'm thinking to move this comments in all the examples to speaker notes and ask audience as questions